playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -100,6 +100,48 @@
     flat: yes
   when: streisand_client_test
 
+# simple-obfs begins here
```

Comment:
Apologies for not flagging this in my first pass, but instead of adding these `# begins here` and `# ends here` comments, how about creating a separate tasks file under `playbooks/roles/shadowsocks/tasks/` called `simple-obfs.yml` - then in `playbooks/roles/shadowsocks/tasks/main.yml` you can include the `simple-obfs.yml` tasks without needing any "begin"/"ends" comments. The simple-obfs stuff will be self-contained in its own tasks file.

Does that sound OK?